github/orgs_codesecurity_configurations_test.go

```diff
@@ -382,7 +382,6 @@ func TestOrganizationsService_SetDefaultCodeSecurityConfiguration(t *testing.T)
 					"code_scanning_default_setup": "enabled"
 				}
 		}`)
-		w.WriteHeader(http.StatusOK)
```

Comment:
The changes to this file seem wrong to me.
I think you can revert the changes to this file.

Reply:
This was a warning which I fixed.

Per https://pkg.go.dev/net/http 

```
If WriteHeader is not called explicitly, the first call to Write
will trigger an implicit WriteHeader(http.StatusOK).
```

So I think we should either keep this change or explicitly call `w.WriteHeader(http.StatusOK)` before we write to `w`. Any thoughts?

Reply:
Oh, interesting, so this is unrelated to this PR.  OK.  I'm fine to keep the change.  Thanks.